data/hai/hai reveal 6 end.txt

```diff
@@ -317,25 +335,45 @@ mission "Hai Reveal [C05] Cleanup"
 			`	She gives you a hard stare, but her resolve isn't up to the task for more than a couple seconds. She looks away, and you can see her blinking away more tears. "Is this to be my end, then?" she asks. "Is this the price I pay for my efforts?"`
 			label decide
 			choice
-				`	(Spare her.)`
-					goto spare
-				`	(End her.)`
-					goto end
 				`	"Am I to be judge, jury, and executioner?"`
 					goto judge
+					to display
+						not "Hai Reveal [C05] Cleanup: label judge"
+				`	(Spare her.) "The whole galaxy hunts you now. Let that be your justice."`
+					goto spare
+				`	(End her.) "The price you pay is justice."`
+					goto end
+				`	(Spare her.) "I've sinned more than you. If people like us decided what is right or wrong, the galaxy would be in cinders."`
+					goto spare
+				`	(End her.) "This is for all the trouble you put me through."`
```

Comment:
I suggest this change because it emphasises that the first two spoken choices are moralising options, whereas the second two are personal options.

I consider this an optional suggestion though.
```suggestion
				`	(End her.) "This is for all the trouble you've caused me, personally."`
```